metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_usage_v2.py

```diff
@@ -363,6 +379,9 @@ def _get_operation_aspect_work_unit(
                     lastUpdatedTimestamp=last_updated_timestamp,
                     actor=user_urn,
                     operationType=operation_type,
+                    customOperationType=query_type
+                    if operation_type is OperationTypeClass.CUSTOM
+                    else None,
```

Comment:
Is there harm in passing this if the operation type class is not `CUSTOM`, for more info? I know we don't do this elsewhere in the code but could start here

Reply:
hmm, I thought that myself as well. Then customOperationType just behaves more like `"nativeOperationType"` . cc - @jjoyce0510  - can you confirm this one as well please ?